File: tests/fixtures/swift/modified_method/after.swift
class Counter {
    var value: Int = 0

    init(value: Int) {
        self.value = value
    }

    func bump() {
        value = step(value)
        self.log()
    }

    private func log() {}
}

func step(_ value: Int) -> Int {
    return value + 1
}

File: tests/fixtures/rust/modified_method/after.rs
pub struct Counter {
    value: u32,
}

impl Counter {
    pub fn new() -> Self {
        Counter { value: 0 }
    }

    pub fn bump(&mut self) {
        self.value = step(self.value);
        self.log();
    }

    fn log(&self) {}
}

fn step(value: u32) -> u32 {
    value + 1
}
